configuration/configuration-mainnet.yaml

```diff
@@ -131,28 +131,25 @@ TraceChainSyncHeaderServer: False
 TraceChainSyncProtocol: False
 
 # Trace DNS Resolver messages.
-TraceDNSResolver: False
+TraceDNSResolver: True
```

Comment:
these changes should probably not go into this configuration file, but be changed when needed. what do you think?
it's hard to guess the effect this has.

Reply:
> these changes should probably not go into this configuration file, but be changed when needed. what do you think?
We should log enough so that there is a chance to troubleshoot network issues experienced by users. 
> it's hard to guess the effect this has.
Which is why I tested it.